Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -20,6 +20,17 @@ internal static class ShardMapUtils
         /// </summary>
         internal static readonly string ConnectRetryCount = "ConnectRetryCount";
 
+        /// <summary>
+        /// SqlConnectionStringBuilder property that allows specifying
+        /// active directoty authentication to connect to SQL instance.
+        /// </summary>
+        internal static readonly string Authentication = "Authentication";
+
+        /// <summary>
+        /// String representation of SqlAuthenticationMethod.ActiveDirectoryIntegrated
+        /// </summary>
```

Comment:
Yes, that's why I am using hardcoded string. I will add above note as a comment.